CodeStarterCamp_Week2/main.swift

```diff
@@ -8,5 +8,27 @@
 
 import Foundation
 
-print("Hello, World!")
+let myNumbers: Set<Int> = [1, 2, 3, 4, 5, 6]
+let winningNumbers: Set<Int> = generateLottoNumbers()
 
+checkLottoNumbers(my: myNumbers, winning: winningNumbers)
+
+func generateLottoNumbers() -> Set<Int> {
+    var winningNumbers: Set<Int> = Set<Int>()
+    while (winningNumbers.count < 6 ) {
```

Comment:
```suggestion
    while winningNumbers.count < 6 {
```

`winningNumbers.count < 6` 자체가 Bool 타입의 값입니다. 왜냐하면 `<`가 `Bool` 타입의 값을 반환하는 함수이기 때문입니다.

그래서 괄호로 감싸줄 필요가 없습니다! 😃